operator/pkg/controlplane/etcd/etcd.go

```diff
@@ -63,7 +63,7 @@ func installKarmadaEtcd(client clientset.Interface, name, namespace string, cfg
 
 	etcdStatefulSetBytes, err := util.ParseTemplate(KarmadaEtcdStatefulSet, struct {
 		StatefulSetName, Namespace, Image, ImagePullPolicy, EtcdClientService string
-		CertsSecretName, EtcdPeerServiceName                                  string
+		CertsSecretName                                                       string
```

Comment:
just a nit, it would be better if `ImagePullPolicy` and `EtcdClientService` could be moved to the same line as `CertsSecretName`.

Reply:
I'm glad you pointed this out! I had a feeling something was off with the spacing 😁, but now everything looks good! :)